management_api_app/.env.sample

```diff
@@ -0,0 +1,37 @@
+# API configuration
+# -----------------
+# DEBUG=True                                    # when debug is set to True, debugging information for unhandled exceptions is shown in the swagger UI and logging is more verbose
+
+# OAUTH Information - client ids etc. for the AAD Apps
+# ----------------
+API_CLIENT_ID=__CHANGE_ME__                     # The AppId for the API service principal (TRE API)
+API_CLIENT_SECRET=__CHANGE_ME__                 # The Client secret fo the TRE API application
+SWAGGER_UI_CLIENT_ID=__CHANGE_ME__              # The AppId for the Swagger service principal (TRE Swagger UI)
+AAD_TENANT_ID=__CHANGE_ME__                     # The Azure AD tenant
+
+# API parameters
+# --------------
+RESOURCE_LOCATION=westeurope                    # The location for deployed resources (this can be set to anything as the deployment service is not called locally)
+TRE_ID=mytre-dev-3142                           # The Azure TRE instance name - used for deployment of resources (can also be set to anything when debuggign locally)
+
+# State store configuration
+# -------------------------
+STATE_STORE_ENDPOINT=https://localhost:8081     # The Cosmos DB endpoint - keep localhost if using an emulator
```

Comment:
```suggestion
STATE_STORE_ENDPOINT=https://localhost:8081     # The Cosmos DB endpoint - keep localhost if using an emulator. Otherwise https://<your_cosmos_db>.documents.azure.com:443/
```